source/extensions/filters/http/aws_lambda/config.h

```diff
@@ -180,13 +180,17 @@ class AWSLambdaRouteConfig : public Router::RouteSpecificFilterConfig {
   }
   bool unwrapAsAlb() const { return unwrap_as_alb_; }
   Transformation::TransformerConstSharedPtr transformerConfig() const { return transformer_config_; }
+  Transformation::TransformerConstSharedPtr requestTransformerConfig() const { return request_transformer_config_; }
   bool hasTransformerConfig() const { return has_transformer_config_; }
+  bool hasRequestTransformerConfig() const { return has_request_transformer_config_; }
```

Comment:
this can be `request_transformer_config_ != nullptr` and `has_request_transformer_config_` can be removed? that way you don't have 2 variables to keep in sync?

Reply:
Resolved this here: https://github.com/solo-io/envoy-gloo/pull/177/commits/c0e235adc9d34b18061c78df7f7a0d393812e668